101-azure-database-migration-service/azuredeploy.json

```diff
@@ -0,0 +1,594 @@
+
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters":
+    {
+		"location":
+		{
+			"defaultValue": "[resourceGroup().location]",
+			"type": "String",
+			"metadata":
+			{
+				"description": "Location where the resources will be deployed."
+			}
+		},
+        "createPublicIP":
+        {
+            "defaultValue": false,
```

Comment:
use true for the sample, as people likely want to see the VM when done

Reply:
Done